packages/calcite-components/.eslintrc.cjs

```diff
@@ -1,23 +1,25 @@
-{
-  "env": {
+module.exports = {
```

Comment:
I was going to originally suggest keeping this as `json` if it didn't break anything, but this will align perfectly with the changes we'll need to make for [future of configuring ESLint](https://eslint.org/docs/latest/use/configure/configuration-files-new). 🎉

Reply:
it needs to be a cjs file so I can use `__dirname`